Formula/libpostal.rb

```diff
@@ -0,0 +1,28 @@
+class Libpostal < Formula
+  desc "C library for parsing/normalizing street addresses around the world"
+  homepage "https://github.com/openvenues/libpostal"
+  url "https://github.com/openvenues/libpostal/archive/v1.0.0.tar.gz"
+  sha256 "3035af7e15b2894069753975d953fa15a86d968103913dbf8ce4b8aa26231644"
+  license "MIT"
+  head "https://github.com/openvenues/libpostal.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "curl" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "./bootstrap.sh"
+    system "./configure", "--datadir=#{share}/libpostal-data",
+                          "--prefix=#{prefix}"
+
+    # {ENV.make_jobs} cannot be used as it only works with exactly 4
+    system "make", "-j4"
+    system "make install"
+  end
+
+  test do
+    File.exist?("#{include}/libpostal/libpostal.h")
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
I struggled with this. This library seems to need like 100GB of ML training data to be functional, so it's not really feasible to test it by calling it. This was the best test I could come up with.

Reply:
If it fails reliably without that data that's also an acceptable test.

Reply:
Ah! That's not a bad idea. I'll do that tomorrow too!